src/cdrom/cdriso.cc

```diff
@@ -429,6 +429,10 @@ bool PCSX::CDRIso::readTrack(const IEC60908b::MSF time) {
     if (!m_cdHandle || m_cdHandle->failed()) {
         return false;
     }
+    
+    if (this->getTrackType(time) == TrackType::CDDA) {
```

Comment:
You should be able to use `m_currentTrack` for this.

Reply:
Oh actually, no, not from this function. I would probably do this in the `readTrack` from cdrom.cc instead.

Reply:
Ah, yeah that makes more sense. Couldn't decide the best place between functions to add this 😅